File: ws/lainez/u1/ws08CD_JavaClasses/ChickenFarmSimulator/javascript/Chicken.js
class Chicken {
    constructor (id, name, color, molting, bornOnDate) {
        this.id = id;
        this.name = name;
        this.color = color;
        this.molting = molting;
        this.bornOnDate = new Date(bornOnDate);
    }

getId(){
    return this.id;
}

setId(id){
    this.id = id;
}
getName(){
    return this.name;
}
setName(name){
    this.name = name;
}

getColor(){
    return this.color;
}
setColor(color){
    this.name = color;
}

getMolting(){
    return this.molting;
}
setMolting(molting){
    this.name = molting;
}

getBornOnDate(){
    return this.bornOnDate;
}

setBornOnDate(bornOnDate){
    this.bornOnDate = new Date(bornOnDate);
}

}

File: ws/mera/u1/WS08 ChickenFarmSimulator/src/main/java/ec/edu/espe/farmsimulator/model/Chicken.java

package ec.edu.espe.farmsimulator.model;

import java.util.Date;

/**
 *
 * @author heiso
 */
public class Chicken {
    private int id;
    private String name;
    private String color;
    private int age;
    private boolean molting;
    private Date bornOnDate;
    
    

    public Chicken(int id, String name, String color, int age, boolean molting, Date bornOnDate) {
        this.id = id;
        this.name = name;
        this.color = color;
        this.age = age;
        this.molting = molting;
         this.bornOnDate = bornOnDate;

    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getColor() {
        return color;
    }

    public int getAge() {
        return age;
    }

    public boolean getMolting() {
        return molting;
    }
    
    public Date getBornOnDate() {
        return bornOnDate;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public void setMolting(boolean molting) {
        this.molting = molting;
    }
    
    public void setBornOnDate(Date bornOnDate) {
        this.bornOnDate = bornOnDate;
    }

    @Override
    public String toString() {
        return "Chicken{" + "id=" + id + ", name=" + name + ", color=" + color + ", age=" + age + ", molting=" + molting + ", bornOnDate=" + bornOnDate + '}';
    }
    
    

}
